launcher/app/src/main/res/layout/activity_multi_user.xml

```diff
@@ -0,0 +1,163 @@
+<?xml version="1.0" encoding="utf-8"?>
+<android.support.constraint.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:background="@drawable/kitkit_background"
+    tools:context=".MultiUserActivity">
+
+    <Button
+        android:id="@+id/Add"
+        android:layout_width="80.16dp"
+        android:layout_height="80.16dp"
+        android:background="@drawable/adduser"
+        android:onClick="AddAllUser"
+        app:layout_constraintBottom_toBottomOf="@+id/addCircle"
+        app:layout_constraintEnd_toEndOf="@+id/addCircle"
+        app:layout_constraintStart_toStartOf="@+id/addCircle"
+        app:layout_constraintTop_toTopOf="@+id/addCircle" />
+
+    <Button
+        android:id="@+id/Select"
+        android:layout_width="0dp"
+        android:layout_height="0dp"
+        android:layout_marginStart="20dp"
+        android:layout_marginLeft="20dp"
+        android:layout_marginTop="20dp"
+        android:layout_marginEnd="20dp"
+        android:layout_marginRight="20dp"
+        android:layout_marginBottom="20dp"
+        android:background="@drawable/selectuser"
+        android:onClick="SelectUser"
+        app:layout_constraintBottom_toBottomOf="@+id/selectCircle"
+        app:layout_constraintDimensionRatio="1:1"
+        app:layout_constraintEnd_toEndOf="@+id/selectCircle"
+        app:layout_constraintStart_toStartOf="@+id/selectCircle"
+        app:layout_constraintTop_toTopOf="@+id/selectCircle"
+        app:layout_constraintVertical_bias="0.0" />
+
+    <TextView
+        android:id="@+id/textView"
+        android:layout_width="wrap_content"
+        android:layout_height="0dp"
+        android:gravity="center"
+        android:text="Add User\nउपयोगकर्ता जोड़ें"
+        android:textColor="@android:color/background_light"
+        android:textSize="37px"
+        android:textStyle="bold"
+        app:layout_constraintBottom_toBottomOf="@+id/textView6"
+        app:layout_constraintEnd_toEndOf="@+id/addCircle"
+        app:layout_constraintHorizontal_bias="0.509"
+        app:layout_constraintStart_toStartOf="@+id/addCircle"
+        app:layout_constraintTop_toTopOf="@+id/textView6" />
+
+    <TextView
+        android:id="@+id/textView6"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="40dp"
+        android:layout_marginBottom="8dp"
+        android:gravity="center"
+        android:text="Select User\nउपयोगकर्ता चयन"
```

Comment:
Please use $#$ as delimiter instead of \n everywhere

Reply:
\n is used for printing Hindi text in next line